packages/react/src/analytics/integrations/GA4/GA4.ts

```diff
@@ -72,6 +73,7 @@ import type {
  * Google Analytics 4 Integration.
  */
 class GA4 extends integrations.Integration<GA4IntegrationOptions> {
+  protected googleConsentMode?: GoogleConsentMode;
```

Comment:
You can use the ! operator to tell typescript that this will be set after the constructor is called.

```suggestion
  protected googleConsentMode!: GoogleConsentMode;
```